annotations/views.py

```diff
@@ -364,6 +366,62 @@ def collection_texts(request, collectionid):
     return HttpResponse(template.render(context))
 
 
+class RecentActivity(models.Model):
```

Comment:
@erickpeirson : the class won't be a database model(mistakenly inherited the class from model class ) but kindof a bean class which helps us to maintain contributions for user.